### ISSUE TITLE
Issue with installation

### ISSUE THREAD

**Issue by awalling** [thackl/gggenomes#95]
Hello,

I am attempting to install gggenomes. When I run the following lines of code, I get the following errors:

`devtools::install_github("thackl/thacklr")
Downloading GitHub repo thackl/thacklr@HEAD
Error: Failed to install 'thacklr' from GitHub:
  Command failed (1)
In addition: Warning message:
In system(full, intern = TRUE, ignore.stderr = quiet) :
  running command ''/usr/bin/git' ls-remote https://git.bioconductor.org/packages/ggtree RELEASE_3_14 2>/dev/null' had status 1

devtools::install_github("thackl/gggenomes")
Downloading GitHub repo thackl/gggenomes@HEAD
Error: Failed to install 'gggenomes' from GitHub:
  Command failed (1)
In addition: Warning message:
In system(full, intern = TRUE, ignore.stderr = quiet) :
  running command ''/usr/bin/git' ls-remote https://git.bioconductor.org/packages/rtracklayer RELEASE_3_14 2>/dev/null' had status 1`

I have also attempted downloading the R package and installing locally, and receive the same error message.

Any assistance would be appreciated!

**Comment by thackl** [comment]
I'm not sure what is going on here. The installations seems to crash while installing bioconductor dependencies (ggtree, rtracklayer). Did you install ggtree through bioconductor as shown in the install instructions before installing gggenomes? Did that work without errors?

**Comment by awalling** [comment]
Hi - yes, I have successfully installed and used ggtree in the past, and loaded the library prior to attempting to install gggenomes. I reinstalled ggtree and then attempted to install gggenomes using devtools again and ran into the same error.

**Comment by iimog** [comment]
I'm happy to have a look at this and help with the installation. In order to do so can you give some information on your setup:
- what operating system (and version) are you using?
- which version of R is installed?
- which version of `devtools` is installed?
- you can answer these questions by providing the output of `library(devtools);sessionInfo()`

What happens if you manually run:
`/usr/bin/git ls-remote https://git.bioconductor.org/packages/ggtree RELEASE_3_14`
on the command line?

**Comment by awalling** [comment]
Hi @iimog , thanks so much for your assistance. For anyone else reading, I'm on a Mac Catalina that I recently updated the OS of, and I think when I did that I needed to update my Xcode developer tools.

When I ran

`/usr/bin/git ls-remote https://git.bioconductor.org/packages/ggtree RELEASE_3_14`

I got the following error message:

`xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun`

Googling brought me to this [stack overflow answer](https://stackoverflow.com/questions/52522565/git-is-not-working-after-macos-update-xcrun-error-invalid-active-developer-pa)

Updating Xcode allowed me to successfully install gggenomes using devtools.

Thank you!
